Xamarin.PropertyEditing/Xamarin.PropertyEditing.csproj

```diff
@@ -20,4 +26,15 @@
       <DependentUpon>Resources.resx</DependentUpon>
     </Compile>
   </ItemGroup>
+
+  <Target Name="GetFilesToSign" BeforeTargets="SignFiles" Condition=" '$(Configuration)' == 'Release' Or '$(Configuration)' == 'ReleaseWindows'">
```

Comment:
Can you explain why this and the other `FileToSign` `ItemGroup` are needed? I thought MicroBuild automatically identified these assemblies. Are you using a different signing certificate or something?

Reply:
`Microsoft400` seems to be the standard cert so I'm not sure why this is needed. You can enable or disable based on configuration in your pipeline.

Reply:
My understanding was each dll inside the vsix needed to be called out in this way. Mostly going off of samples and guidance provided through out wiki.


@mattnorflus , do we not actually need to do it this way?

Reply:
I think that's only for nested content. For this csproj's assembly, it should be signed automatically by the microbuild targets. But @mattnorflus should know better than I do.

Reply:
https://dev.azure.com/devdiv/DevDiv/_wiki/wikis/DevDiv.wiki/27099/Guidance-for-Xamarin-MicroBuild-Signing-Migration?anchor=handling-nested-content

Reply:
This is correct, the `FilesToSign` `ItemGroup` is how MicroBuild knows which files to sign in the SignFiles target. Each file that ends up in the vsix must be signed in this way prior to getting packaged into the vsix.

Reply:
And then there's also [this guidance](https://dev.azure.com/devdiv/DevDiv/_wiki/wikis/DevDiv.wiki/27099/Guidance-for-Xamarin-MicroBuild-Signing-Migration?anchor=avoid-oversigning-if-building-multiple-configurations) for avoiding signing during non-Release builds, but I feel like that is more easily solved by not installing the signing pluging for non-Release builds in your pipeline.

Reply:
For additional clarification, the [core MicroBuild guidance](https://dev.azure.com/devdiv/DevDiv/_wiki/wikis/DevDiv.wiki/650/MicroBuild-Signing?anchor=modify-your-project-file-to-include-%60filestosign%60-itemgroup) states that the `FilesToSign` `ItemGroup` is required for all files that need to be sign. "In order to add signing to your projects, modify your project files to add the following item group"

Reply:
Hmm, interesting, maybe it's `Microsoft.VisualStudio.Internal.MicroBuild.VisualStudio` that automatically populates `FilesToSign`? We definitely don't set it ourselves in the editor build.

Reply:
So if I'm understanding correctly, it sounds like this is expected, at least for the version of MicroBuild we're using.

Reply:
I guess so. It seems the internal package does a bit more heavy lifting automatically.